roles/ceph-mon/tasks/docker/fetch_configs.yml

```diff
@@ -11,7 +10,7 @@
       - /var/lib/ceph/bootstrap-mds/ceph.keyring
 
 - name: stat for ceph config and keys
-  local_action: stat path={{ item }}
+  local_action: stat path={{ playbook_dir }}/{{ fetch_directory }}/{{ fsid }}/{{ item }}
```

Comment:
I'm afraid fsid will be empty since ceph-common role is never called.
So where do you get it from?